### ISSUE TITLE
The Llama-2 example does not work now

### ISSUE THREAD

**Issue by sunny0826** [skypilot-org/skypilot#2680]
Run the [Llama-2 example](https://github.com/skypilot-org/skypilot/tree/master/llm/llama-2) according to the documentation and got an error:

```bash
None of PyTorch, TensorFlow >= 2.0, or Flax have been found. Models won't be available and only tokenizers, configuration and file/data utilities can be used.
(task, pid=6905) Traceback (most recent call last):
(task, pid=6905)   File "/anaconda/envs/chatbot/lib/python3.9/runpy.py", line 197, in _run_module_as_main
(task, pid=6905)     return _run_code(code, main_globals, None,
(task, pid=6905)   File "/anaconda/envs/chatbot/lib/python3.9/runpy.py", line 87, in _run_code
(task, pid=6905)     exec(code, run_globals)
(task, pid=6905)   File "/anaconda/envs/chatbot/lib/python3.9/site-packages/fastchat/serve/model_worker.py", line 35, in <module>
(task, pid=6905)     import torch
(task, pid=6905) ModuleNotFoundError: No module named 'torch'
```

**Additional information**

- Cloud: Azure
- Region: australiaeast
- Config: `chatbot-hf.yaml`

**Comment by concretevitamin** [comment]
Thanks for the report @sunny0826. https://github.com/skypilot-org/skypilot/blob/master/llm/llama-2/chatbot-hf.yaml uses A100:1 which isn't offered by Azure. To reproduce, what's the gpus / resources config that you tried on Azure?

**Comment by Michaelvll** [comment]
This should be fixed by #2825. Closing it now.
